admin-dev/themes/new-theme/package-lock.json

```diff
@@ -4161,8 +4161,7 @@
                 "ansi-regex": {
```

Comment:
@NeOMakinG when I did my npm install, "optional: true" was added in many places in the package-lock.json, do you think it's ok ?

Reply:
It should not do anything dangerous, I think it's not a problem at all

Reply:
thanks for the deedback :)